server/project.go

```diff
@@ -196,11 +196,16 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 			defer resp.Body.Close()
 
 			if resp.StatusCode >= 300 {
-				return echo.NewHTTPError(http.StatusInternalServerError,
-					fmt.Sprintf("Failed to create webhook for project ID: %v, status code: %d",
-						repositoryCreate.ProjectID,
-						resp.StatusCode,
-					))
+				reason := fmt.Sprintf(
+					"Failed to create webhook for project ID: %d, status code: %d",
+					repositoryCreate.ProjectID,
+					resp.StatusCode,
+				)
```

Comment:
nit: mind attach the github issue as a comment here? just to provide more context.